### PR TITLE
add tests for TableNamesMapper (#26217)

### LINKED ISSUE #26217: Add unit test for TableNamesMapper public method
Hi community,

This issue is for https://github.com/apache/shardingsphere/discussions/28487.

## Aim

This issue is to add unit tests for the TableNamesMapper class, it includes the following subtasks:

- [ ]  TableNamesMapper#contains
- [ ]  TableNamesMapper#getTableNames
- [ ]  TableNamesMapper#put
- [ ]  TableNamesMapper#remove

You can refer this pr - https://github.com/apache/shardingsphere/pull/25386 to add unit test.

## Basic Qualifications
- Java
- Maven
- Junit

### PR DESCRIPTION
**Pull request by KonarzewskiP**
Fixes #26217.

Changes proposed in this pull request:
  - Add tests for:
- [x]  TableNamesMapper#contains
- [x]  TableNamesMapper#getTableNames
- [x]  TableNamesMapper#put
- [x]  TableNamesMapper#remove

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [x] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: infra/common/src/test/java/org/apache/shardingsphere/infra/rule/identifier/type/TableNamesMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    @BeforeEach
+    void setUp() {
+        tableNamesMapper = new TableNamesMapper();
+    }
+
+    @Test
+    void assertContainsTable() {
+        tableNamesMapper.put("foo_table");
+        assertTrue(tableNamesMapper.contains("foo_table"));
```
Comment (strongduanmu) [inline infra/common/src/test/java/org/apache/shardingsphere/infra/rule/identifier/type/TableNamesMapperTest.java:40]: Can you test a case insensitive scenario?

File: infra/common/src/test/java/org/apache/shardingsphere/infra/rule/identifier/type/TableNamesMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ */
+
+package org.apache.shardingsphere.infra.rule.identifier.type;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
```
Comment (strongduanmu) [inline infra/common/src/test/java/org/apache/shardingsphere/infra/rule/identifier/type/TableNamesMapperTest.java:25]: Please do not use import *

File: infra/common/src/test/java/org/apache/shardingsphere/infra/rule/identifier/type/TableNamesMapperTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    void assertContainsTable() {
+        tableNamesMapper.put("foo_table");
+        assertTrue(tableNamesMapper.contains("foo_table"));
+    }
+
+    @Test
+    void assertGetTableNames() {
+        tableNamesMapper.put("foo_table_1");
+        tableNamesMapper.put("foo_table_2");
+        assertThat(tableNamesMapper.getTableNames(), hasItems("foo_table_1", "foo_table_2"));
```
Comment (strongduanmu) [inline infra/common/src/test/java/org/apache/shardingsphere/infra/rule/identifier/type/TableNamesMapperTest.java:47]: Do you think assertThat(xxx, is) is better. This helps the code maintain a uniform style.

File: infra/common/src/test/java/org/apache/shardingsphere/infra/rule/identifier/type/TableNamesMapperTest.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        assertFalse(tableNamesMapper.contains("foo_table_1"));
+        assertThat(tableNamesMapper.getTableNames(), hasSize(0));
+    }
+
+    @Test
+    void assertPut() {
+        tableNamesMapper.put("foo_table");
+        assertThat(tableNamesMapper.getTableNames(), hasSize(1));
+    }
+}
```
Comment (strongduanmu) [inline infra/common/src/test/java/org/apache/shardingsphere/infra/rule/identifier/type/TableNamesMapperTest.java:64]: Please add new at the end of file.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

**Comment by strongduanmu** [comment]
Hi @KonarzewskiP, thank you for your contribution. For the beginner, it's better to look at code conduct first.

**Comment by KonarzewskiP** [comment]
Hi @strongduanmu ,

Thank you very much for your time to check my PR and your comments. I just pushed a new PR with changes. Please have a look when you have time.

**Comment by strongduanmu** [comment]
> Hi @strongduanmu ,
> 
> Thank you very much for your time to check my PR and your comments. I just pushed a new PR with changes. Please have a look when you have time.

Hi @KonarzewskiP, can you take a look at ci error?

**Comment by KonarzewskiP** [comment]
Hi @strongduanmu 

It looks like all checks have passed. Have a look when you have time, please. Thank you!

**Review by strongduanmu — APPROVED** [review]
LGTM, merged.

**Pull request merged**